tests/integration/review_app/test_app_df_graph.py

```diff
@@ -149,4 +155,10 @@ def update_graph(rows, selected_rows):
 
     dash_duo.start_server(app)
     dash_duo.wait_for_element("#waitfor")
+
+    # Wait for table
+    WebDriverWait(dash_duo.driver, _TIMEOUT).until(
+        EC.presence_of_element_located((By.CSS_SELECTOR, "#{}".format(IDS["table"])))
+    )
```

Comment:
Why not just `dash_duo.wait_for_element("#{}".format(IDS["TABLE"])`?

Reply:
🙈